lifecycle-management/ahk-lifecycle-management/Properties/serviceDependencies.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "appInsights1": {
+      "type": "appInsights"
+    },
+    "storage1": {
+      "type": "storage",
+      "connectionId": "AzureWebJobsStorage"
+    }
+  }
+}
```

Comment:
Másik projektben nincs ilyen fájl, enélkül nem tud működni?

Reply:
Projekt létrehozáskor került bele, enélkül is működik. Töröltem.